bootstrap.go

```diff
@@ -48,16 +48,23 @@ const (
 		Delete_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Create_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Drop_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Process_priv		ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Process_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
```

Comment:
If you open it locally, you'll find them all aligned by three tabs.
Seems github can't show the ident correctly.
Should I change those tabs to whitespaces?